napari/components/_viewer_key_bindings.py

```diff
@@ -141,6 +144,42 @@ def transpose_axes(viewer: Viewer):
     viewer.dims.transpose()
 
 
+@register_viewer_action(trans._('Rotate layers 90 degrees counter-clockwise.'))
+def rotate_layers(viewer: Viewer):
+    if viewer.dims.ndisplay == 3:
+        show_info(trans._('Rotating layers only works in 2D.'))
```

Comment:
I've confirmed manually that this works but it would be good to add coverage for it. You could mock `show_info` as in [this test](https://github.com/napari/napari/blob/main/napari/_qt/_qapp_model/_tests/test_file_menu.py#L151), with a 3D layer loaded and ndisplay set to 3.

Reply:
so ViewerModel doesn't emit `show_info` so I just test to ensure the rotation matrix has not changed (rotate is disabled). Is that good enough? Or should I look for a place to add a full viewer test, rather than ViewerModel.